CHANGELOG.md

```diff
@@ -7,7 +7,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 ### Added
-  * [#186](https://github.com/jhedstrom/DrupalDriver/issues/168) Provide a method to directly authenticate on Drupal 8.
+  * [#186](https://github.com/jhedstrom/DrupalDriver/issues/18§) Provide a method to directly authenticate on Drupal 8.
```

Comment:
I don't think this is the intended change...it's also to the `CHANGELOG.md` file, not the `README.md` as noted in the summary...

Reply:
```suggestion
  * [#186](https://github.com/jhedstrom/DrupalDriver/issues/186) Provide a method to directly authenticate on Drupal 8.
```

Reply:
Hi @jhedstrom, my point was that there's a wrong number #168, while it should be #186.